Neos.Flow/Classes/Property/TypeConverter/PersistentObjectConverter.php

```diff
@@ -205,6 +206,12 @@ public function convertFrom($source, $targetType, array $convertedChildPropertie
                     throw new InvalidTargetException($exceptionMessage, 1421498771);
                 }
             }
+            $propertyType = TypeHandling::getTypeForValue($propertyValue);
+            $isCollectionType = TypeHandling::isCollectionType($propertyType);
+            //$isAggregateRelation = TODO: Check if element type is aggregate root
```

Comment:
We could try to detect AR-relations and avoid the exception here. But then again, if that relation has a manual `orphanRemoval`, the issue would be the same.
```suggestion
```